CONTRIBUTING.md

```diff
@@ -43,40 +43,53 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
+To ensure all your commits are signed, you may choose to add this alias to your global `.gitconfig`:
+
+_~/.gitconfig_
 
-*~/.gitconfig*
 ```
 [alias]
   amend = commit -s --amend
   cm = commit -s -m
   commit = commit -s
 ```
+
 Or you may configure your IDE, for example, Visual Studio Code to automatically sign-off commits for you:
 
 <a href="./assets/images/git-signoff-vscode.png" ><img src="./assets/images/git-signoff-vscode.png" width="50%"><a>
 
 ### Set Up Your Development Environment
-Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also install Ruby Version Manager [RVM](https://rvm.io) - a command-line tool which allows you to easily install, manage and work with multiple ruby environments on your local machine. Alternatively, if you're running Windows 10, update to version 1903 Build 18362 or higher. You can also shift to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+- First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler.
+
+**Note:** Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine.
+Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:
+
+- [WSL1](https://docs.microsoft.com/en-us/windows/wsl/install-win10) for Windows build version 1607 or higher.
 
 ### <a name="flow">Contribution Flow</a>
+
 Please contribute! The Layer5 site uses GitHub Pages (Jekyll) to host its site. The process of contributing to documentation follows this flow:
 
 1. Fork a copy of this repo.
 1. Get a local clone of your fork of the site.
-`git clone https://github.com/layer5io/layer5`
+   `git clone https://github.com/layer5io/layer5`
 1. Create and checkout a new branch to make changes within
-`git checkout -b <my-username>/<my-changes>` 
+   `git checkout -b <my-username>/<my-changes>`
 1. Edit site/add content.
-`vi <specific page>.md` # or use your favorite IDE
+   `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
-`make site` # this will run a local web server with "live reload" conveniently enabled.
+   `make site` # this will run a local web server with "live reload" conveniently enabled.
+
+- **Note:** _From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when deploying for production based on differences of the handling of trailing slashes between Jekyll and GitHub Pages._
```

Comment:
Hi, a collaborator reviewed this line in the other repo and requested that it be changed. Hope you don’t mind changing it to this ”There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artefacts for production.”

Reply:
Ok let me get to it